vktrace/vktrace.md

```diff
@@ -18,6 +18,7 @@ Options for the `vktrace` command are:
 | -tr&nbsp;&lt;string&gt;<br>&#x2011;&#x2011;TraceTrigger&nbsp;&lt;string&gt; | Start/stop trim by hotkey or frame range. String arg is one of:<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hotkey-[F1-F12\|TAB\|CONTROL]<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;frames-&lt;startframe&gt;-&lt;endframe&gt;| on |
 | -tpp&nbsp;&lt;bool&gt;<br>&#x2011;&#x2011;TrimPostProcessing&nbsp;&lt;bool&gt; | Enable trim post-processing to make trimmed trace file smaller, see description of VKTRACE_TRIM_POST_PROCESS below | false |
 | -v&nbsp;&lt;string&gt;<br>&#x2011;&#x2011;Verbosity&nbsp;&lt;string&gt; | Verbosity mode - "quiet", "errors", "warnings", or "full" | errors |
+| -tbs&nbsp;&lt;string&gt;<br>&#x2011;&#x2011;TrimBatchSize&nbsp;&lt;string&gt; | Set the maximum trim commands batch size per command buffer  |  ~device memory allocation limit divide by 100 |
```

Comment:
Should reference the environment variable in the description of the -tbs option (similar to what is done for the -tpp option).Then add a section under "Trace Tools Environment Variables" describing the environment variable. The description of the environment variable should include why the user might want to set this option. (And mention that this applies to vktrace/vktrace layer, which should have been done for the other environment variable descriptions. I think all the variables apply to vktrace/vktrace layer, not to replay, so maybe change the heading and the introduction paragraph instead.)

Change "~device memory allocation limit divide by 100" to "device memory allocation limit divided by 100"